src/main/java/com/redhat/labs/omp/resource/EngagementResource.java

```diff
@@ -173,6 +178,23 @@ public Response findCustomers(@NotBlank @QueryParam("suggest") String match) {
         return Response.ok(customerSuggestions).build();
     }
 
+    @GET
+    @Path("/categories")
+    @SecurityRequirement(name = "jwt", scopes = {})
+    @APIResponses(value = { 
+            @APIResponse(responseCode = "401", description = "Missing or Invalid JWT"),
+            @APIResponse(responseCode = "200", description = "Customer data has been returned.") })
+    @Operation(summary = "Returns customers list")
+    public List<Category> getAllCategories(@QueryParam("suggest") String match) {
+
+        if(null == match || "".equals(match.trim())) {
```

Comment:
i think (null || match.isBlank()) would be simpler

Reply:
agreed and updated.